LBH.AdultSocialCare.Api/V1/Gateways/CarePackages/Concrete/CarePackageHistoryGateway.cs

```diff
@@ -38,9 +39,17 @@ public async Task<IEnumerable<CarePackageHistoryDomain>> ListAsync(Guid carePack
             return carePackageHistory;
         }
 
-        public void Create(CarePackageHistory newCarePackageHistory)
+        public async Task Create(CarePackageHistory newCarePackageHistory)
         {
-            _dbContext.CarePackageHistories.Add(newCarePackageHistory);
+            await _dbContext.CarePackageHistories.AddAsync(newCarePackageHistory);
+            try
+            {
+                await _dbContext.SaveChangesAsync();
+            }
+            catch (Exception ex)
+            {
+                throw new DbSaveFailedException($"Could not save care package history to database {ex.Message}");
+            }
```

Comment:
Could you do the save in use case where history is being added and leave save out of gateway? The save changes in use case should save added history as well

Reply:
make sense, miss that!